.github/workflows/ms.network.virtualnetworkgateways.yml

```diff
@@ -106,8 +106,7 @@ jobs:
       - name: 'Using test file [${{ matrix.moduleTestFilePaths }}]'
         uses: ./.github/actions/templates/validateModuleDeployment
         with:
-          templateFilePath: '${{ env.modulePath }}/deploy.bicep'
-          parameterFilePath: '${{ env.modulePath }}/${{ matrix.moduleTestFilePaths }}'
+          templateFilePath: '${{ env.modulePath }}/${{ matrix.moduleTestFilePaths }}'
```

Comment:
Just a comment to ask: should #1844  be considered first?

Reply:
If it's pipeline would not be red, sure 😄

Reply:
It appears only the removal failed there. Will re-run it. If the pipeline turns out green I'd vote to merge that first

Reply:
Converted this PR to draft until the other is done, conflicts are resolved & the pipeline was re-run.

Reply:
Incorporated the changes. The PR is again ready for review